gapir/cc/memory_manager.cpp

```diff
@@ -22,74 +22,42 @@
 #include <utility>
 #include <vector>
 
-//                 mMemory[0]
```

Comment:
This picture is still effectively valid. This is still how everything is laid out.

Reply:
Reintroduced. I removed it because it wasn't true anymore for the system allocator backed cache.